### ISSUE TITLE
static-site: apostrophe vs. &apos; bugs

### ISSUE THREAD

**Issue by jhannah** [nigelhorne/ged2site#117]
My 21 times great-grandfather `Patrick A'Hannay` seems to be causing problems. 😄
* The text of `static-site/Patrick-A'Hannay-1250.html` looks good.
  * But the Family Tree (the GraphViz `img/I4183.png`) lists the husband and wife as:
    * `Lady Agnes A&apos;Hannay`
    * `Patrick A&apos;Hannay`
![Screenshot 2024-06-05 at 18 38 23](https://github.com/nigelhorne/ged2site/assets/62206/f7a90b46-86d4-44d8-910e-f81a7ec84ba9)

* `static-site/surnames.html` looks good, links to `static-site/A'hannay.html`, but that's 404 
  * because that file was actually written to disk as `static-site/A&apos%3bhannay.html`
    * whose links are good, but the text is broken, listing:
      * `Lady A&apos;Hannay (n&eacute;e A&apos;Hannay) 1260 -`
      * `Patrick A&apos;Hannay 1250 -`

```
./ged2site -cFdh 'Jay Weston Hannah' -l ~/Desktop/jay_new.ged
```

**Comment by nigelhorne** [comment]
There are two problems here, the 404 error and the Graphviz issue.

Please try the attached commit for the 404 error.  The Graphviz issue may take more thinking.

**Comment by nigelhorne** [comment]
Please review, I believe I have fixed everything however I may have missed something.

**Comment by jhannah** [comment]
Looks like you fixed several of these, but #112 is back and my ugly fixes blank out the whole page so I can't see if GraphViz is fixed yet...

**Comment by jhannah** [comment]
This is working now. Woot! My fork branch `112-and-115` 98af16deccef62925d85c9221c4b61a7f2eef971 output `static-site/I4183.html`.
